fission/src/systems/physics/PhysicsSystem.ts

```diff
@@ -165,9 +165,7 @@ class PhysicsSystem extends WorldSystem {
      * The pause works off of a request counter.
      */
     public ReleasePause() {
-        if (this._pauseCounter > 0) {
-            this._pauseCounter--
-        }
+        if (this._pauseCounter > 0) this._pauseCounter--
```

Comment:
Can we push this even further?

```suggestion
        if (this._pauseCounter) this._pauseCounter--
```